test/format/ops.c

```diff
@@ -1314,6 +1405,12 @@ wts_read_scan(TABLE *table, void *arg)
         if (keyno > max_rows)
             keyno = max_rows;
 
+        if (GV(OPS_BOUND_CURSOR) && mmrand(NULL, 1, 10) == 1) {
+            /* Reset the position of the cursor, so that we can apply bounds on the cursor. */
+            testutil_check(cursor->reset(cursor));
```

Comment:
It might be best to clear the bounds, rather than reset (see other comment).  But -- if the code is always clearing the bounds whenever it is set, maybe this is not needed?

Reply:
Currently range cursors has a restriction where the user is not allowed to place bounds on a cursor that is currently positioned. wts_read_scan does point in time searches through out to verify the contents of the table, but never resets the position, therefore I have manually resetted it so that we can place bounds.